manifests/reverse_proxy.pp

```diff
@@ -37,7 +37,7 @@
   apache::vhost { $vhost_name:
     ensure                 => $ensure,
     servername             => $certs::apache::hostname,
-    aliases                => $certs::apache::cname,
+    serveraliases          => $certs::apache::cname,
```

Comment:
this was a bug, should it be an own PR (which would fail tests, but oh well)?

Reply:
Now I've cherry picked this, but I don't have a changelog entry. So in hindsight, it would have been nice.

Reply:
well… ;)

Reply:
In https://github.com/theforeman/puppet-foreman_proxy_content/pull/430 I tried to describe it.